.vscode/settings.json

```diff
@@ -128,4 +128,8 @@
     "unordered_set": "cpp",
     "algorithm": "cpp"
   },
+  "rust-analyzer.linkedProjects": [
```

Comment:
please remove this / move to another PR, I don't think its needed? My local is ok without it